pkg/controller/addon/klusterlet_addon_controller.go

```diff
@@ -32,6 +34,24 @@ const (
 	annotationValues = "addon.open-cluster-management.io/values"
 )
 
+const (
+	// AnnotationKlusterletDeployMode is the annotation key of klusterlet deploy mode
+	AnnotationKlusterletDeployMode = "import.open-cluster-management.io/klusterlet-deploy-mode"
+
+	// AnnotationEnableHostedModeAddons is the key of annotation which indicates if the add-ons will be enabled
+	// in hosted mode automatically for a managed cluster
+	AnnotationEnableHostedModeAddons = "addon.open-cluster-management.io/enable-hosted-mode-addons"
+
+	// AnnotationKlusterletHostingClusterName is the annotation key of hosting cluster name for klusterlet
+	AnnotationKlusterletHostingClusterName = "import.open-cluster-management.io/hosting-cluster-name"
+
+	// AnnotationAddOnHostingClusterName is the annotation key of hosting cluster name for add-ons
+	AnnotationAddOnHostingClusterName = "addon.open-cluster-management.io/hosting-cluster-name"
+)
+
+var hostedAddOns = sets.NewString(agentv1.PolicyFrameworkAddonName, agentv1.ConfigPolicyAddonName,
```

Comment:
consider making it common and available in multiple places?

Reply:
Created a separated file for constants.

Reply:
ok, Based on https://github.com/stolostron/klusterlet-addon-controller/pull/171/files#r1031941384, if there is no other place to use this, it is unnecessary to put it into a separate file.